### PR TITLE
Project crud features

### PR DESCRIPTION
**Pull request by WilliamOna**
added functionality and service/controller tests for getAll, getById, updateById, deleteById, and updateById

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
+		</dependency>
+	</dependencies>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.springframework.boot</groupId>
+				<artifactId>spring-boot-maven-plugin</artifactId>
```
Comment (RevatureGentry) [inline pom.xml:66]: Can we add `rev_checks` Checkstyle Configuration to this project? 

[Reference](https://github.com/RevatureGentry/rev_checks)

**Review by RevatureGentry — CHANGES_REQUESTED** [review]
Everything looks good - please add in configuration for the style guide then we can merge in 🔥

File: pom.xml (hunk 1 above)
Comment (WilliamOna) [reply pom.xml:66]: can do. what are the ${puppycrawl-checkstyle.version} and the ${checkstyle-maven-plugin.version} values? the show up as errors

Comment (RevatureGentry) [reply pom.xml:66]: This should get you there 👍 
https://github.com/RevatureGentry/rev_checks/blob/master/pom.xml#L16

Comment (WilliamOna) [reply pom.xml:66]: I added rev_checks to the project

**Review by RevatureGentry — APPROVED** [review]
LGTM 👍

**Pull request merged**